doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -0,0 +1,71 @@
+Lagrangian Physical Properties
+----------------------------------
+In this subsection, the physical properties of the simulation are defined. These properties include gravitational acceleration (``gx``, ``gy`` and ``gz``), ``number of particle types``, and for each type, particle ``diameter``, particle ``density``, ``Young's modulus`` of particle and wall, ``Poisson ratio`` of particle and wall, ``restitution coefficient`` of particle and wall, ``friction coefficient`` of particle and wall and ``rolling friction coefficient`` of particle and wall.
```

Comment:
```suggestion
In this subsection, the physical properties of the particles are defined. These properties include gravitational acceleration (``gx``, ``gy`` and ``gz``), ``number of particle types``, and for each type, particle ``diameter``, particle ``density``, ``Young's modulus`` of particle and wall, ``Poisson ratio`` of particle and wall, ``restitution coefficient`` of particle and wall, ``friction coefficient`` of particle and wall and ``rolling friction coefficient`` of particle and wall.
```

Reply:
I would add a more general description on the top and explain each parameter below.